enhancements/workload-partitioning/pao-render.md

```diff
@@ -0,0 +1,190 @@
+replaced with `-`.
+
+To get started with this template:
+1. **Pick a domain.** Find the appropriate domain to discuss your enhancement.
+1. **Make a copy of this template.** Copy this template into the directory for
+   the domain.
+1. **Fill out the "overview" sections.** This includes the Summary and
+   Motivation sections. These should be easy and explain why the community
+   should desire this enhancement.
+1. **Create a PR.** Assign it to folks with expertise in that domain to help
+   sponsor the process.
+1. **Merge at each milestone.** Merge when the design is able to transition to a
+   new status (provisional, implementable, implemented, etc.). View anything
+   marked as `provisional` as an idea worth exploring in the future, but not
+   accepted as ready to execute. Anything marked as `implementable` should
+   clearly communicate how an enhancement is coded up and delivered. If an
+   enhancement describes a new deployment topology or platform, include a
+   logical description for the deployment, and how it handles the unique aspects
+   of the platform. Aim for single topic PRs to keep discussions focused. If you
+   disagree with what is already in a document, open a new PR with suggested
+   changes.
+1. **Keep all required headers.** If a section does not apply to an
+   enhancement, explain why but do not remove the section. This part
+   of the process is enforced by the linter CI job.
+
+The `Metadata` section above is intended to support the creation of tooling
+around the enhancement process.
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The Performance addon operator is a day 2 optional operator installed via OLM. However the [Workload partitioning](management-workload-partitioning.md) feature needs to have some configuration ready at the installation time when PAO is not yet ready.
+This feature is about adding a `render` mode to PAO that would allow an admin to pre-compute all the Openshift manifests that are needed in a way that prevents any typing mistakes from happening.
+These generated manifests can then be passed to the installer and later taken over by the PAO reconcile loop. 
+
+## Motivation
+
+There are many pieces that must be configured just right for the latency and resource management to work properly. PAO addresses that for day 2 latency tuning, but there was no way to use PAO's knowledge at the install time.
+
+### Goals
+
+1) PAO can execute outside of a cluster via podman/docker, consume user manifests from a directory and generate additional manifests to another (or the same) directory
```

Comment:
I don't have an issue with saying that we would use a container image, but it feels like a level of detail that fits better in the Implementation section, rather than in the Goals. Placing it here narrows our options in a way that doesn't feel necessary. I might rewrite this as

> PAO can be run outside of a cluster to generate manifests to pass to the OpenShift installer.

Reply:
Well, you are right, but I think it should be part of the design. How would the doc writers know what the expected user flow looks like?

Reply:
If we talk about the delivery mechanism down in either the "Proposal" or "Implementation Details/Notes/Constraints" section, all of the information is still available for the tech writers.